yarn.lock

```diff
@@ -2357,9 +2357,9 @@ ajv@^6.1.0, ajv@^6.10.0, ajv@^6.10.2, ajv@^6.12.4, ajv@^6.12.5:
     uri-js "^4.2.2"
 
 ajv@^8.0.1:
```

Comment:
I'm not sure why we need this lib, can u check and make sure we did not install some libs/packages we are not using?

Reply:
I can't find where this package is used, but it seems like it's very popular it has 53 million downloads a week. 
I think it might be part of CRA since you can use Json files throughout the project.
I checked the package.json file, it seems fine. I removed the dependencies that belong to the dev dependencies (they were in both places). Maybe we have too many eslint plugins, but I'll probably use them once I configure the eslint.json. Should I remove them now?

Reply:
We can uninstall these packages and test whether the app works or not without them?

Reply:
![image](https://user-images.githubusercontent.com/68707570/142731769-30bcffa7-f605-46f4-8a4f-b20f566cd57b.png)
This is what I get if I try to remove it